src/Core/Managed/Shared/Extensibility/Implementation/External/Base_types.cs

```diff
@@ -0,0 +1,60 @@
+
+//------------------------------------------------------------------------------
+// This code was generated by a tool.
+//
+//   Tool : Bond Compiler 0.4.1.0
+//   File : Base_types.cs
+//
+// Changes to this file may cause incorrect behavior and will be lost when
+// the code is regenerated.
+// <auto-generated />
+//------------------------------------------------------------------------------
+
+
+// suppress "Missing XML comment for publicly visible type or member"
+#pragma warning disable 1591
+
+
+#region ReSharper warnings
+// ReSharper disable PartialTypeWithSinglePart
+// ReSharper disable RedundantNameQualifier
+// ReSharper disable InconsistentNaming
+// ReSharper disable CheckNamespace
+// ReSharper disable UnusedParameter.Local
+// ReSharper disable RedundantUsingDirective
+#endregion
+
+namespace Microsoft.ApplicationInsights.Extensibility.Implementation.External
+{
+    using System.Collections.Generic;
+
+    
+    
+    [System.CodeDom.Compiler.GeneratedCode("gbc", "0.4.1.0")]
+    internal partial class Base
```

Comment:
gbc does, but powershell script cleans them up so we don't have extra code to compile extra dependencies to Bond assembly